apps/meteor/client/views/omnichannel/currentChats/useAllCustomFields.tsx

```diff
@@ -0,0 +1,9 @@
+import { useEndpoint } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export const useAllCustomFields = () => {
```

Comment:
You may try `UseQueryResult<T>`.

```suggestion
export const useAllCustomFields = (): UseQueryResult<OperationResult<'GET', '/v1/livechat/custom-fields'>> => {
```